packages/trpc/server/routers/viewer/_router.tsx

```diff
@@ -1,71 +1,146 @@
-import app_Basecamp3 from "@calcom/app-store/basecamp3/trpc-router";
-import app_RoutingForms from "@calcom/app-store/routing-forms/trpc-router";
-import { userAdminRouter } from "@calcom/features/ee/users/server/trpc-router";
-import { featureFlagRouter } from "@calcom/features/flags/server/router";
-import { insightsRouter } from "@calcom/features/insights/server/trpc-router";
-
 import { mergeRouters, router } from "../../trpc";
-import { loggedInViewerRouter } from "../loggedInViewer/_router";
-import { publicViewerRouter } from "../publicViewer/_router";
-import { timezonesRouter } from "../publicViewer/timezones/_router";
-import { adminRouter } from "./admin/_router";
-import { apiKeysRouter } from "./apiKeys/_router";
-import { appsRouter } from "./apps/_router";
-import { attributesRouter } from "./attributes/_router";
-import { authRouter } from "./auth/_router";
-import { availabilityRouter } from "./availability/_router";
-import { bookingsRouter } from "./bookings/_router";
-import { deploymentSetupRouter } from "./deploymentSetup/_router";
-import { domainWideDelegationRouter } from "./domainWideDelegation/_router";
-import { dsyncRouter } from "./dsync/_router";
-import { eventTypesRouter } from "./eventTypes/_router";
-import { googleWorkspaceRouter } from "./googleWorkspace/_router";
-import { highPerfRouter } from "./highPerf/_router";
-import { oAuthRouter } from "./oAuth/_router";
-import { viewerOrganizationsRouter } from "./organizations/_router";
-import { paymentsRouter } from "./payments/_router";
-import { routingFormsRouter } from "./routing-forms/_router";
-import { slotsRouter } from "./slots/_router";
-import { ssoRouter } from "./sso/_router";
-import { viewerTeamsRouter } from "./teams/_router";
-import { webhookRouter } from "./webhook/_router";
-import { workflowsRouter } from "./workflows/_router";
 
-export const viewerRouter = mergeRouters(
-  loggedInViewerRouter,
+let _viewerRouter: ReturnType<typeof mergeRouters> | null = null;
+
+export async function getViewerRouter() {
```

Comment:
- Only used in `createAppRouter()` (which is only used in `ssgInit()` in `ssg.ts` at the moment). `ssgInit()` is also not used anywhere atm.
- Hence, technically, we can remove both `ssgInit()` and this file but maybe we will need this in the future for SSG support